keyboards/handwired/polykybd/split72/keymaps/default/keymap.c

```diff
@@ -186,6 +186,7 @@ typedef struct _poly_sync_t {
     layer_state_t default_ls;
     uint16_t last_latin_kc;
     uint32_t crc32;
+    uint8_t unicode_mode;
```

Comment:
we have to add new data filed after the crc32 as I calculate it over a length of size of the struct - sizeof(crc32)

Reply:
doh, yeah I read the transfer code and I knew this, I just didn't think about it, I'll move it

Reply:
Well maybe it is a good time to write code comments :)

Reply:
added ;)